base/dcp_feed_type.go

```diff
@@ -161,12 +152,13 @@ type SGFeedIndexParams struct {
 
 // cbgtFeedParams returns marshalled cbgt.DCPFeedParams as string, to be passed as feedparams during cbgt.Manager init.
 // Used to pass basic auth credentials and xattr flag to cbgt.
-func cbgtFeedParams(ctx context.Context, spec BucketSpec, scope string, collections []string, dbName string) (string, error) {
-	feedParams := &SGFeedSourceParams{}
-	feedParams.DbName = dbName
-
-	if spec.UseXattrs {
-		feedParams.IncludeXAttrs = true
+func cbgtFeedParams(ctx context.Context, scope string, collections []string, dbName string) (string, error) {
+	feedParams := &SGFeedSourceParams{
+		DbName: dbName,
+		DCPFeedParams: cbgt.DCPFeedParams{
+			AutoReconnectAfterRollback: true,
+			IncludeXAttrs:              true,
```

Comment:
This change isn't related to the PR (to always set xattrs=true), and it's hard to tell the impact of this change.  I'm assuming that elsewhere we're not using cbgt in the first place if we're not using xattrs?  How confident are you that there aren't unintended side-effects here?